### ISSUE TITLE
Updated to GameMaker Beta v2024.400.0.516, all sound effects have wrong associations now

### ISSUE THREAD

**Issue by Jonnil** [YoYoGames/GameMaker-Bugs#5113]
### Description

After updating to GameMaker Beta v2024.400.0.516, sound effects in my game are playing incorrectly. It appears that the IDs associated with the sound effects are being scrambled or offset during gameplay. As an example, when a sound effect labeled "Sound A" should be played, it's actually "Sound B" that is heard. This issue persists despite verifying that the correct sound files are assigned in the asset tree within GameMaker.

Attempted solutions include replacing the affected sound files with duplicates of themselves, as well as deleting and re-importing all sound files.
The issue persists across both PC and Nintendo Switch platforms that I'm developing for.
Asset cache has been cleared using the "Clean" function (Ctrl + F7), but the problem persists.
A comparison video has been recorded demonstrating correct sound effect playback in GameMaker Stable v2024.2.0.132 versus the incorrect playback in GameMaker Beta v2024.400.0.516: [https://youtu.be/aYOXhBVHnZE?si=s8md0Q1QHK2NrMZF](https://youtu.be/aYOXhBVHnZE?si=s8md0Q1QHK2NrMZF)

The issue appears to be specific to the latest Beta version of GameMaker, as sound effects play correctly in the stable version. This suggests a potential bug introduced in the Beta release.

I first documented this bug in this GameMaker Community forum post: [https://forum.gamemaker.io/index.php?threads/updated-to-gamemaker-beta-v2024-400-0-516-all-sound-effects-have-wrong-associations-now.110747/](https://forum.gamemaker.io/index.php?threads/updated-to-gamemaker-beta-v2024-400-0-516-all-sound-effects-have-wrong-associations-now.110747/)

Here is the Google Drive link to my "Contact Us Package", the file was too big to attach in this description: [https://drive.google.com/file/d/1WmLzHOlYAnMQAHKqvoWc0a45BjAzLOPo/view](https://drive.google.com/file/d/1WmLzHOlYAnMQAHKqvoWc0a45BjAzLOPo/view)

Here is the GitHub for my project: [https://github.com/Jonnil/Sticky-Paws-DX](https://github.com/Jonnil/Sticky-Paws-DX)

### Expected Change

Fix audio bug where sound effects are playing incorrectly, as if they've been scrambled

### Steps To Reproduce

1. Update to GameMaker Beta v2024.400.0.516.
2. Play the game with sound effects implemented.
3. Observe that the sound effects are playing incorrectly, with IDs seemingly swapped or scrambled.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.400 (Betas)

### Which platform(s) are you seeing the problem on?

Windows, Switch

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by backYard321** [comment]
I ran into this too - I was told this might be triggered by a blank audio asset being present in the project, which I have. Do you also have one?

**Comment by Jonnil** [comment]
I don't have any blank audio assets in the asset tree. But now that you mention this, I have noticed that some audio files are not playing when using audio_create_stream. I have voice clips for the player character that should be playing, .ogg files from the Included Files folder. For some reason, they are not loading in, there is no sound. Does this also count as a blank audio asset then?
But why aren't the .ogg files being loaded when I'm using audio_create_stream? I can still hear level music correctly, which is using audio_create_stream. It's just the player character that isn't loading the .ogg voice clips at all.

**Comment by Jonnil** [comment]
I finally got voice clips working again. Somehow the path name was incorrect, so that was a mistake on my end. The voice clips are all playing correctly when I load them with audio_create_stream now, but all the sound effects in the asset tree are still not working in-game. I don't know what sound file could be considered blank, and thus causing this problem, if that is indeed the problem here.
But this problem I'm having, wouldn't it still be considered a bug in the GameMaker Beta v2024.400? When loading the project in a stable version of GameMaker. I tried playing my game in GameMaker Stable v2024.2.0.132, and all the sound effects worked like normal then. This problem is only happening in the latest Beta version for me.
